common-queries/src/clue/scala/queries/common/ObsQueriesGQL.scala

```diff
@@ -134,7 +145,15 @@ object ObsQueriesGQL:
       }
     """
 
+    object Scalars:
+      type SignalToNoise = lucuma.core.math.SignalToNoise
```

Comment:
This shouldn't be necessary, since it's declared in the schema (in `lucuma-schemas`).